src/components/pages/dashboard.tsx

```diff
@@ -37,8 +51,42 @@ import {
   PaginationNext,
   PaginationPrevious,
 } from '@/components/molecules/pagination.tsx'
+import { useGetProductsByVendorQuery } from '@/lib/redux/features/product/api.ts'
+import { useCommonStore } from '@/lib/zustand/common.ts'
+
+import { Checkbox } from '../atoms/checkbox.tsx'
 
 export default function DashboardPage() {
+  const { products, isSuccess } = useGetProductsByVendorQuery(
+    { id: '2550' },
+    {
+      selectFromResult: result => ({
+        ...result,
+        products: result.data?.products,
+      }),
+    },
+  )
```

Comment:
Is the usage of `id: 2550` intended (as a temporary solution)?

Reply:
Yah, should be integrated with taking the path param but not yet implemented. So it is just temporary.